### PR TITLE
added WebsiteJsonLd component

### PR DESCRIPTION
**Pull request by pweikel**
Added structured data component to homepage

### CODE REVIEW

File: @ndlib/gatsby-theme-marble/src/components/Shared/Seo/SeoContent/index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { GatsbySeo } from 'gatsby-plugin-next-seo'
+import WebsiteJsonLd from './WebsiteJsonLd'
```
Comment (danthewolfe) [inline @ndlib/gatsby-theme-marble/src/components/Shared/Seo/SeoContent/index.js:4]: Should this be on every page or just the index and/or search pages?

Comment (pweikel) [reply @ndlib/gatsby-theme-marble/src/components/Shared/Seo/SeoContent/index.js:4]: No. This should only be on the homepage.

File: @ndlib/gatsby-theme-marble/src/components/Shared/Seo/SeoContent/WebsiteJsonLd/test.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import WebsiteJsonLd from './'
+import { JsonLd } from 'gatsby-plugin-next-seo'
+
+test('Website JSON LD schema on homepage', () => {
+  const wrapper = shallow(<WebsiteJsonLd pathname='/' siteUrl='https://marble.nd.edu' />)
+  console.log(wrapper.debug())
```
Comment (danthewolfe) [inline @ndlib/gatsby-theme-marble/src/components/Shared/Seo/SeoContent/WebsiteJsonLd/test.js:8]: remove console.log

File: @ndlib/gatsby-theme-marble/src/components/Shared/Seo/SeoContent/index.js (hunk 1 above)
Comment (danthewolfe) [reply @ndlib/gatsby-theme-marble/src/components/Shared/Seo/SeoContent/index.js:4]: Unless I'm misreading this, it appears to get placed on every page.So this should likely on in marble-web on the index page only.

**Review by danthewolfe — APPROVED** [review]

**Pull request merged**